### DISCUSSION TITLE
gp.exe

### DISCUSSION THREAD

**Discussion by navidun** [martinpaljak/GlobalPlatformPro#301] [General]
hi. when do you release the gp.exe of version v21.12.31?

**Comment by Kyahzworld** [comment]
My Javacard jcop is locked can anyone helo

**Comment by martinpaljak** [comment]
What do you mean by "locked" ? If you don't know the keys, you're out of luck. Please also see https://github.com/martinpaljak/GlobalPlatformPro/wiki/Keys

**Comment by navidun** [comment]
Hi Martinpaljak, when do you release the gp.exe of version v21.12.31? i want to install applet on SSD but i don't know how to do it.
please help me with gp.exe version v21.12.31

**Comment by martinpaljak** [comment]
@navidun GPPro v24.10.15 was released on the given date.
